app/uk/gov/hmrc/tradergoodsprofiles/models/errors/ErrorResponse.scala

```diff
@@ -98,3 +98,7 @@ case class InvalidErrorsResponse(correlationId: String, errors: Option[Seq[Error
       )
     )
 }
+
+//object InvalidErrorsResponse {
```

Comment:
why this is commented out? Do you not need that?

Reply:
removed